### PR TITLE
``cat``: print logs into stderr

### LINKED ISSUE #624: tt cat should write logs into stderr not stdout
Subj
```
❯ 2>/dev/null tt cat 00000000000000000000.snap --format json | head -10
   • Running cat with files: [00000000000000000000.snap]

• Result of cat: the file "00000000000000000000.snap" is processed below •
{"HEADER":{"lsn":387,"group_id":1,"type":"RAFT","timestamp":1696258559.5658},"BODY":{"0":1}}
{"HEADER":{"lsn":388,"type":"PROMOTE","timestamp":1696258559.5658},"BODY":{"2":0,"3":0,"83":0}}
```

`Running cat` and `Result of cat` must be printed into stderr so `tt cat` can be unix-friendly

### PR DESCRIPTION
**Pull request by sssciel**
Now ``tt cat`` prints logs into stderr not stdout.

Closes #624

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -610,7 +610,7 @@ Common description. For a detailed description, use `tt help command` .
 -   `check` - check an application file for syntax errors.
 -   `connect` - connect to the tarantool instance.
 -   `rocks` - LuaRocks package manager.
--   `cat` - print into stdout the contents of .snap/.xlog files.
+-   `cat` - print into stderr the contents of .snap/.xlog files.
```
Comment (psergee) [inline README.md:613]: `tt cat` should still print snap/xlog file content to stdout, not stderr.

File: cli/checkpoint/checkpoint.go [hunk 2]
```diff
@@ -45,7 +46,7 @@ func Cat(tntCli cmdcontext.TarantoolCli) error {
 	scanner := bufio.NewScanner(stdoutPipe)
 	scanner.Split(bufio.ScanLines)
 	for scanner.Scan() {
-		fmt.Println(scanner.Text())
+		fmt.Fprintf(os.Stderr, scanner.Text())
```
Comment (psergee) [inline cli/checkpoint/checkpoint.go:49]: The content must be printed to stdout. Only diagnostic messages must go to stderr.

File: README.md (hunk 1 above)
Comment (sssciel) [reply README.md]: fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ stdout/stderr and `tt` logs go to `tt.log` file.
 - Ignore app-instance delimiters for Tarantool 3.0 instances.
 - Don't use dash as an app-instance delimiter. At the same time, `cartridge_app-stateboard`
 treated as a special case.
+- ``tt cat``: ``Result of cat`` now prints into stderr not stdout.
```
Comment (psergee) [inline CHANGELOG.md:31]: Can we not be so specific about the text? `tt cat`: all diagnostic messages are printed to stderr.

File: cli/checkpoint/lua/cat.lua [hunk 4]
```diff
@@ -217,4 +217,4 @@ local function main()
 end
 
 main()
-os.exit(0)
+os.exit(0)
```
Comment (psergee) [inline cli/checkpoint/lua/cat.lua:220]: Why is new line removed?

File: cli/checkpoint/checkpoint.go [hunk 5]
```diff
@@ -23,36 +24,23 @@ type Opts struct {
...
 	stdinPipe, err := cmd.StdinPipe()
 	if err != nil {
 		return err
 	}
 	stdinPipe.Write([]byte(catFile))
 	stdinPipe.Close()
 
-	if err := cmd.Start(); err != nil {
-		return fmt.Errorf("result of cat: %s", errbuff.String())
+	if err := cmd.Run(); err != nil {
```
Comment (psergee) [inline cli/checkpoint/checkpoint.go:38]: `Run` waits for the command to complete. There is no need to keep the `Wait` call below.

File: cli/checkpoint/checkpoint.go [hunk 6]
```diff
@@ -23,36 +24,23 @@ type Opts struct {
...
+		return fmt.Errorf("result of cat: %v", err)
```
Comment (psergee) [inline cli/checkpoint/checkpoint.go:39]: Use %w to wrap the error.

File: cli/checkpoint/lua/cat.lua (hunk 4 above)
Comment (sssciel) [reply cli/checkpoint/lua/cat.lua:220]: idk what is it

**Comment by psergee** [comment]
In the commit message and the PR description:
- print -> prints
- PR description mentions integration test was added. But I don't see added integration tests.

**Review by psergee — APPROVED** [review]
LGTM. Pls, rebase

**Pull request merged**